layouts/partials/footer.html

```diff
@@ -1,21 +1,43 @@
 {{ $copy := site.Copyright }}
 {{ $year := now.Year }}
 {{ $logo := printf "img/logos/%s" site.Params.logos.footer | relURL }}
-<footer class="footer has-background-black">
+<footer class="footer has-background-white">
   <div class="container">
     <div class="columns">
       <div class="column">
         <img class="is-footer-logo" src="{{ $logo }}" alt="{{ site.Title }} footer logo">
       </div>
 
       <div class="column">
-        
+        <p class="is-size-6 has-text-black has-text-weight-light">
+          <a class="has-text-black is-small" target="_blank" href="{{ site.Params.github }}">
+            <i class="fab fa-github"></i>&nbsp;GitHub
```

Comment:
Instead of `&nbsp;`, I recommend doing it like this for all of the logos:

```html
<a class="has-text-black is-small" target="_blank" href="{{ site.Params.github }}">
  <span class="icon">
    <i class="fab fa-github"></i>
  </span>
  <span>
    GitHub
  </span>
</a>
```

Reply:
Good idea, changing.